acvm-repo/acir_field/src/field_element.rs

```diff
@@ -405,6 +415,51 @@ mod tests {
         assert_eq!(max_num_bits_bn254, 254);
     }
 
+    proptest! {
+        #[test]
+        fn test_endianness_prop(value in any::<u64>()) {
+            let field = FieldElement::<ark_bn254::Fr>::from(value);
+            
+            // Test serialization consistency
+            let le_bytes = field.to_le_bytes();
+            let be_bytes = field.to_be_bytes();
+            
+            let mut reversed_le = le_bytes.clone();
+            reversed_le.reverse();
+            prop_assert_eq!(be_bytes, reversed_le, "BE bytes should be reverse of LE bytes");
```

Comment:
Maybe try comparing `&be_bytes` and `&reversed_le`. You can run `cargo clippy --all-targets --workspace --locked` to see if it has anything to complain about.